README.md

```diff
@@ -4,6 +4,23 @@ Please see the main [PostHog docs](https://posthog.com/docs).
 
 Specifically, the [Ruby integration](https://posthog.com/docs/integrations/ruby-integration) details.
 
+## Developing Locally
+
+1. Install Ruby (and optionally `rbenv` to control ruby versions)
+2. Install Bundler
+3. Run `bundle install` to install dependencies
+
+## Running example file
+
+1. Build the `posthog-ruby` gem by calling: `gem build posthog-ruby.gemspec`.
+2. Install the gem locally: `gem install ./posthog-ruby-<version>.gem`
+3. Run `ruby example.rb`
+
+## Testing
+
+1. Run `bundle exec rspec`
+2. An example of running specific tests: `bundle exec rspec spec/posthog/client_spec.rb:26`
```

Comment:
Thank you for documenting this. I think it will make the project more welcoming to new people. 🙌